perses/tests/testsystems.py

```diff
@@ -307,7 +307,7 @@ def __init__(self, **kwargs):
         from pkg_resources import resource_filename
         valence_xml_filename = resource_filename('perses', 'data/amber99sbildn-valence-only.xml')
         system_generators['vacuum'] = SystemGenerator([valence_xml_filename],
-            forcefield_kwargs={ 'nonbondedMethod' : app.NoCutoff, 'implicitSolvent' : None, 'constraints' : None },
+            forcefield_kwargs={ 'nonbondedMethod' : app.NoCutoff, 'implicitSolvent' : None, 'constraints' : app.HBonds },
```

Comment:
Is this something you also plan to add to other test systems? 

Also, you probably want to make sure your test still runs when you're NOT using constraints.